scripts/rnnlm/change_vocab.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Copyright 2018  Xiaohui Zhang
+
+# This script prepares a new rnnlm-dir commpatible with a new vocab from a provided word-list,
+# given an exisiting rnnlm-dir containing a trained rnnlm. Basically, we copy the feature 
+# embedding, a trained rnnlm and some config files from the old rnnlm-dir. And then we re-
+# generate the unigram_probs.txt (a fixed unigram prob is assigned to words out of the orignal vocab),
+# word_feats.txt and word embeddings.
+
+cmd=run.pl
+oov_unigram_prob=0.01
```

Comment:
This seems kind of high... more like 10^-6 is what I'd choose.   I don't know how critical this is though.